vm-mock/vm.js

```diff
@@ -774,6 +775,12 @@ export default function createMockedABI(
         const hash = sha3.keccak256.arrayBuffer(data);
         return newArrayBuffer(hash);
       },
+      assembly_script_set_chain_id(value) {
+        chainIdMock = value;
+      },
+      assembly_script_chain_id() {
+        return chainIdMock;
```

Comment:
maybe you can return a default value, in case the mock as not been initialized?

Reply:
The default value is set to the chain id of the Buildnet